CHANGELOG.md

```
@@ -16,22 +8,21 @@
-* chores: Add error handling for the curl command by @gitworkflows in <https://github.com/khulnasoft/reconpoint/pull/1367>
-* Update Github Actions Workflows by @khulnasoft in <https://github.com/khulnasoft/reconpoint/pull/1369>
-* chores: Fix docker build on master by @khulnasoft in <https://github.com/khulnasoft/reconpoint/pull/1373>
+* remove redundant docker environment variables by @jxdv in https://github.com/khulnasoft/reconpoint/pull/1353
+* fix: reconPoint installation issue due to orjson and langchain #1362 by @khulnasoft in https://github.com/khulnasoft/reconpoint/pull/1363
+* #1364 Fix whois lookup and improve performance by executing various modules of whois lookup to run concurrently by @khulnasoft in https://github.com/khulnasoft/reconpoint/pull/1368
+* chores: Add error handling for the curl command by @gitworkflows in https://github.com/khulnasoft/reconpoint/pull/1367
+* Update Github Actions Workflows by @khulnasoft in https://github.com/khulnasoft/reconpoint/pull/1369
+* chores: Fix docker build on master by @khulnasoft in https://github.com/khulnasoft/reconpoint/pull/1373
 
 #### New Contributors
+* @gitworkflows made their first contribution in https://github.com/khulnasoft/reconpoint/pull/1367
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)